pymc3/distributions/discrete.py

```diff
@@ -447,7 +447,8 @@ def __init__(self, lower, upper, *args, **kwargs):
     def _random(self, lower, upper, size=None):
         # This way seems to be the only to deal with lower and upper
         # as array-like.
-        samples = stats.uniform.rvs(lower, upper - lower - np.finfo(float).eps,
+        samples = stats.uniform.rvs(lower,
```

Comment:
We really should be using `randint` here, rather than rounding a continuous sample.

Reply:
agree, it should be replaced with `st.randint.rvs(low, high+1)`